xc7/tests/ff_sr_ce/generate.py

```diff
@@ -1,13 +1,16 @@
 import argparse
```

Comment:
I know this is a short script, but a single line comment describing what it does would be useful.